templates/comments_add_visitor.html

```diff
@@ -35,7 +35,7 @@
     -->
 </script>
 
-<hr name="addcomment">
+<hr id="addcomment">
 <form id="post" name="post" action="comments.php" method="post" onsubmit="return chkFormular()" class="form-horizontal">
```

Comment:
I think that this is wrong, because the name is used as anchor in urls #addcomment